### ISSUE TITLE
No module named 'pwd'

### ISSUE THREAD

**Issue by Leon-Huang** [dbcli/mycli#890]
![image](https://user-images.githubusercontent.com/16815164/89197732-c2731780-d5de-11ea-85f3-ec4cf23815d7.png)

```
PS C:\Users\Leon> pip freeze
mycli==1.22.1
PyMySQL==0.10.0
```

OS: win 10
shall I still use an older  version of some libraries?

**Comment by amjith** [comment]
I just released a new version (1.22.2) to workaround this issue. 

Can you please try upgrading? 

```
pip install -U mycli
```

**Comment by Leon-Huang** [comment]
done, after upgrade, mycli==1.22.2 can connect to MySQL
good job, thanks
